security/crowdsec/src/etc/rc.syshook.d/start/50-crowdsec

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# https://docs.opnsense.org/development/backend/autorun.html
+
+/usr/local/opnsense/scripts/OPNsense/CrowdSec/hub-upgrade.sh
```

Comment:
this happens post-crowdsec service start (priority 20). if this is ok please disregard the comment

Reply:
yes it's the upgrade of the parser/scenario collection